### ISSUE TITLE
RStudio latest preview can not knit on Apple silicon MacBook - Bad CPU

### ISSUE THREAD

**Issue by gsgxnet** [rstudio/rstudio#12280]
### System details

    RStudio Edition :  Desktop
    RStudio Version :  Version 2022.12.0-preview+308 (2022.12.0-preview+308)
    OS Version      :  macOS Monterey 12.6 on M2 MacBook Air
    R Version       :  4.2.1

### Steps to reproduce the problem
- open RStudio,
- create a new R Mark-down file with deafault content
- knit it (preview button)

### Describe the problem in detail
`render` fails at start:
```
sh: /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc: Bad CPU type in executable
Error in strsplit(info, "\n")[[1]] : subscript out of bounds
Calls: <Anonymous> ... pandoc_available -> find_pandoc -> lapply -> FUN -> get_pandoc_version
In addition: Warning message:
In system(paste(shQuote(path), "--version"), intern = TRUE) :
  running command ''/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc' --version' had status 126
Execution halted
```
verification in shell:
`/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc --version`  
`zsh: bad CPU type in executable: /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc`  

### Describe the behavior you expected
`render` should produce a html document

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gsgxnet** [comment]
> @gsgxnet Thank you for raising this! Do you know if this worked with earlier versions of the 2022.12.0 builds?
> 
@ronblum I had the same issue with the 2022.12.0 273 daily, dtd October, 18th. Before stating this issue I first wanted to make sure, it is still in the most current daily. Therefore I had installed the newer 308 preview. 
Any other daily you would suggest to try?

**Comment by jonvanausdeln** [comment]
@ronblum , works fine on my M1 machine.

**Comment by gsgxnet** [comment]
May be my system behaves as it does because I do not have Rosetta installed here, pure arm64 Monterey.

Another probably connected issue - I can not create any `quarto` document on the same system:
![image](https://user-images.githubusercontent.com/8518156/199849639-91e93472-6e07-46d7-9c93-ab4ddeec1ca0.png)
Reproducible, just File -> New File -> Quarto Document

**Comment by ronblum** [comment]
@gsgxnet Did you by chance install a version of Quarto from https://quarto.org? If so, it takes precedence over the version that comes bundled with RStudio. If so, either
* download a new version from https://quarto.org, or
* remove `/usr/local/bin/quarto` which will allow RStudio to rely on its bundled version.

**Comment by gsgxnet** [comment]
@ronblum There is no `quarto` on this system from  https://quarto.org/. 
I had installed the R `quarto` package after the problems I have reported here to find out if it might help. It does not, so removed that package again. 
Only `quarto` is the one contained in the RStudio:
```
sudo find / -name quarto -ls 2>&-                  
Password:
1518194        0 drwxr-xr-x    4 gsmaa            admin                 128 Nov  2 21:08 /System/Volumes/Data/Applications/RStudio.app/Contents/Resources/app/quarto
1518217       16 -rwxr-xr-x    1 gsmaa            admin                5556 Nov  2 14:33 /System/Volumes/Data/Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto
1520595        0 drwxr-xr-x    9 gsmaa            admin                 288 Nov  2 21:08 /System/Volumes/Data/Applications/RStudio.app/Contents/Resources/app/quarto/share/lua-types/quarto
1522588        0 drwxr-xr-x    5 gsmaa            admin                 160 Nov  2 21:08 /System/Volumes/Data/Applications/RStudio.app/Contents/Resources/app/quarto/share/extensions/quarto
1518194        0 drwxr-xr-x    4 gsmaa            admin                 128 Nov  2 21:08 /Applications/RStudio.app/Contents/Resources/app/quarto
1518217       16 -rwxr-xr-x    1 gsmaa            admin                5556 Nov  2 14:33 /Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto
1520595        0 drwxr-xr-x    9 gsmaa            admin                 288 Nov  2 21:08 /Applications/RStudio.app/Contents/Resources/app/quarto/share/lua-types/quarto
1522588        0 drwxr-xr-x    5 gsmaa            admin                 160 Nov  2 21:08 /Applications/RStudio.app/Contents/Resources/app/quarto/share/extensions/quarto
```
Same files are shown twice, I assume macOS sets some internal links.  

I have no clue at all what is causing this.

**Comment by gsgxnet** [comment]
I checked with the `file` command:  
`file /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc`  
`/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc: Mach-O 64-bit executable x86_64`  
this is definitely an Intel app, neither Universal nor Apple chips. This is the reason for the Bad CPU error I got. Are you sure you do deliver a fully Apple silicon RStudio  app?  
Anyhow I will try the most current daily now, may by the problem is already solved.

**Comment by kevinushey** [comment]
> May be my system behaves as it does because I do not have Rosetta installed here, pure arm64 Monterey.

Is installing Rosetta an option for you?

**Comment by gsgxnet** [comment]
> Is installing Rosetta an option for you?  

@kevinushey - technically installing Rosetta could be a solution. But I prefer not to do so, because there seems to be no way to get totally rid of it later. And I prefer to avoid clutter if possible. 

There is no real urgent need to get RStudio Univeral running on my new Apple Macbook. I still have an other option to use RStudio. I will patiently wait for a solution. 
Creating a fully Universal app seams not to be so easy, JetBrains has some X86 only components in their apps hiding in corners as well.

**Comment by gsgxnet** [comment]
Retested on Version 2023.03.0+386. Still the same error when Rosetta is not installed.

**Comment by gsgxnet** [comment]
Rechecked with current RStudio and also looking for a local Quarto install.  
Both still need Rosetta2. 
I still hope in the not too distant future there will be an Apple Silicon native Quarto both from quarto.org and embedded in RStudio.  
For Linux there is a ARM variant.

**Comment by kevinushey** [comment]
The next Quarto release should have arm64 builds available; hopefully that will come sometime in the new year.

**Comment by leo-maxwell** [comment]
I came across this problem and the solution is simple: you just download the latest `pandoc` arm64 version and overwrite the old one in the quarto folder inside the `Rstudio` app bundle. The knit worked just fine after this operation. Hope it will help everyone concerned about this problem. I found my laptop to have a much longer battery life after I removed Rosetta 2, and I do not want to install it again.
